### PR TITLE
(#826) Add docs for email notifications

### PR DESCRIPTION
**Pull request by vexx32**
## Description Of Changes

- Update deployments docs to mention email / website notifications for 0.11.0 (and also some that never got added after 0.10.0)

## Motivation and Context

Keeping docs up to date w/ new functionality

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [x] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

#826

### CODE REVIEW

File: input/en-us/central-management/usage/website/deployments.md [hunk 1]
```diff
@@ -287,6 +287,28 @@ This action will mark any completed Deployment plan as archived, and it will no
...
+## Deployment Status Notifications
+
+> :choco_info: **Note**
+>
+> Email notifications require that the Email / SMTP settings have been configured first.
+> If these settings have not been configured, email notifications will not be sent.
+
+### Deployment Completion Notifications
+
+As of Chocolatey Central Management v0.11.0, all deployments will issue an email notification when they finish to the user that created the deployment plan.
```
Comment (st3phhays) [inline input/en-us/central-management/usage/website/deployments.md:299]:
```suggestion
As of Chocolatey Central Management v0.11.0, all Deployment Plans will issue an email notification when they finish to the user that created the Deployment Plan.
```

File: input/en-us/central-management/usage/website/deployments.md [hunk 2]
```diff
@@ -287,6 +287,28 @@ This action will mark any completed Deployment plan as archived, and it will no
...
+The email notification will indicate the status of the deployment, as well as linking back to the deployment details page for further information.
+
+![Example email from Chocolatey Central Managment for a successfully completed deployment, showing the status, start time, and finish time, as well as the deployment name and a link to the deployment.](/assets/images/deployments/ccm-deployments-email-success.png)
+
+### Notification for Scheduled Deployments Unable to Start
+
+As of Chocolatey Central Management v0.10.0, a scheduled deployment that fails to start will issue a push notification as well as an email notification to the user who initially created the deployment plan.
```
Comment (st3phhays) [inline input/en-us/central-management/usage/website/deployments.md:306]:
```suggestion
As of Chocolatey Central Management v0.11.0, a scheduled Deployment Plan that fails to start will issue a push notification as well as an email notification to the user who initially created the Deployment Plan.
```

Wasn't sure if that was 0.10.0 or 0.11.0

File: input/en-us/central-management/usage/website/deployments.md [hunk 3]
```diff
@@ -287,6 +287,28 @@ This action will mark any completed Deployment plan as archived, and it will no
...
+
+> :choco_info: **Note**
+>
+> Email notifications require that the Email / SMTP settings have been configured first.
+> If these settings have not been configured, email notifications will not be sent.
+
+### Deployment Completion Notifications
+
+As of Chocolatey Central Management v0.11.0, all deployments will issue an email notification when they finish to the user that created the deployment plan.
+The email notification will indicate the status of the deployment, as well as linking back to the deployment details page for further information.
```
Comment (st3phhays) [inline input/en-us/central-management/usage/website/deployments.md:300]:
```suggestion
The email notification will indicate the status of the Deployment Plan, as well as linking back to the Deployment Plan details page for further information.
```

File: input/en-us/central-management/usage/website/deployments.md (hunk 1 above)
Comment (st3phhays) [inline input/en-us/central-management/usage/website/deployments.md:299]:
```suggestion
As of Chocolatey Central Management v0.11.0, all Deployment Plans will issue an email notification when they finish to the user that created the deployment plan.
```

**Review by st3phhays — CHANGES_REQUESTED** [review]
I believe we are going after "Deployment Plan" in new docs. I know this document has a mixture of usage in it though.. we'll need to update that at a later time.

File: input/en-us/central-management/usage/website/deployments.md (hunk 2 above)
Comment (vexx32) [reply input/en-us/central-management/usage/website/deployments.md:306]: this one's 0.10.0, we just never actually added docs for it lol

File: input/en-us/central-management/usage/website/deployments.md (hunk 1 above)
Comment (vexx32) [reply input/en-us/central-management/usage/website/deployments.md]: fixed, ty! :3

File: input/en-us/central-management/usage/website/deployments.md (hunk 2 above)
Comment (vexx32) [reply input/en-us/central-management/usage/website/deployments.md]: fixed the rest, ty! :3

File: input/en-us/central-management/usage/website/deployments.md (hunk 3 above)
Comment (vexx32) [reply input/en-us/central-management/usage/website/deployments.md]: fixed!

File: input/en-us/central-management/usage/website/deployments.md (hunk 1 above)
Comment (vexx32) [reply input/en-us/central-management/usage/website/deployments.md]: this one's also good now I think

**Review by corbob — APPROVED** [review]
LGTM

**Review by st3phhays — APPROVED** [review]

**Pull request merged**